README.md

```diff
@@ -88,6 +88,21 @@ gitforce() {
 }
 ```
 
+Or, if you are using ZSH for your shell, add this to your `.zshrc` file:
+
+```zsh
+gitforce() {
+    read "confirm?You are about to commit a potential secret. Are you sure (y/n)?"
+        echo    # (optional) move to a new line
+    if [[ "$confirm" =~ ^[Yy]$ ]]
+    then
+        git config --local hooks.gitleaks false
+        git commit -m "$@" 
+        git config --local hooks.gitleaks true
```

Comment:
does this currently assure that we'll run this line if the commit fails? if not,  could it be made to do so?

Reply:
How about 8e62b8505a8babe0f537ad285180010a070210f3?

it's a bit brute force but should ensure the script continues if `git commit` fails

Reply:
seems reasonable enough to me